app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java

```diff
@@ -34,14 +36,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View rootView = inflater.inflate(R.layout.fragment_main, container, false);
         Button circleButton = (Button) rootView.findViewById(R.id.circleButton);
         Button getHelpNowButton = (Button) rootView.findViewById(R.id.getButton);
-        Button reportingProcessButton = (Button) rootView.findViewById(R.id.reportButton);
+        Button safetyToolsButton = (Button) rootView.findViewById(R.id.safetyToolsButton);
         Button safetyResourceButton = (Button) rootView.findViewById(R.id.safetyResourceButton);
         Button getHelpButton = (Button) rootView.findViewById(R.id.getHelpButton);
 
-        reportingProcessButton.setOnClickListener(new View.OnClickListener() {
+        safetyToolsButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                startActivity(new Intent(getActivity(), HomeScreen.class));
+                //Swapping Safety Tools HomeFragment Screen into the fragment container
+                Fragment safetyToolsFragment = new SafetyToolsFragment();
+                MainActivity.swapFragmentIn(getActivity(),safetyToolsFragment,SafetyToolsFragment.TAG,true);
```

Comment:
All these changes have arised due to change in the function which takes a boolean value as an argument